run_interactive_multiple_host.py

```diff
@@ -43,8 +43,6 @@ def create_engine():
       quantize_kv=FLAGS.quantize_kv_cache,
       max_cache_length=FLAGS.max_cache_length,
       sharding_config=FLAGS.sharding_config,
-      shard_on_batch=FLAGS.shard_on_batch,
```

Comment:
Maybe merge mistake? Why there 2 flags are gone?

Reply:
Intended, to simplify the ray disaggregated work,  not support shard_on_batch and ragged attention in ray multiple for now. Will support later.

Reply:
Make sense.